src/main/java/com/cdancy/bitbucket/rest/features/BranchApi.java

```diff
@@ -114,4 +119,36 @@ Branch getDefault(@PathParam("project") String project,
     @GET
     BranchModel model(@PathParam("project") String project,
                       @PathParam("repo") String repo);
+
+    @Named("branch:list-branch-permission")
+    @Documentation({"https://developer.atlassian.com/static/rest/bitbucket-server/4.14.1/bitbucket-ref-restriction-rest.html#idm45354011023456"})
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("/branch-permissions/2.0/projects/{project}/repos/{repo}/restrictions")
+    @Fallback(BitbucketFallbacks.BranchPermissionPageOnError.class)
+    @GET
+    BranchPermissionPage listBranchPermission(@PathParam("project") String project,
+                                             @PathParam("repo") String repo,
+                                             @Nullable @QueryParam("start") Integer start,
+                                             @Nullable @QueryParam("limit") Integer limit);
+
+    @Named("branch:update-branch-permission")
+    @Documentation({"https://developer.atlassian.com/static/rest/bitbucket-server/4.14.1/bitbucket-ref-restriction-rest.html#idm45354011023456"})
+    @Path("/branch-permissions/2.0/projects/{project}/repos/{repo}/restrictions")
+    @Produces("application/vnd.atl.bitbucket.bulk+json")
```

Comment:
Is this content-type required? Will the already defined `@Produces(MediaType.APPLICATION_JSON)` not work?